src/helics/application_api/Federate.hpp

```diff
@@ -771,8 +772,16 @@ received
     @param configString  the location of the file or config String to load to generate the
     interfaces
     */
-    void registerFilterInterfaces(const std::string& configString);
-
+    void registerConnectorInterfaces(const std::string& configString);
```

Comment:
The Filter method was in place before Translators were invented so the name is outdated.  But part of the public API, so didn't want to remove it.  It will likely be deprecated in the next major release.